documentation/Database-Backups.md

```diff
@@ -0,0 +1,54 @@
+# Performing Manual Database Backups and Restores
+
+## Backup Database
+1. Connect to Database Pod
+2. Open Data management tool to the table such as Sql Server Management Studio (SSMS) or Azure Data Studio (ADS)
+3. On SSMS 
+   <ul>

```

Comment:
<ul> in markdown can usually be replaced with tic marks.
```
1. Bullet
        - List item
        - List item
        - List item
```
Bold is `** around text`.

Reply:
All cleaned up